jaseci_kit/jaseci_kit/modules/fasttext/tests/test_fast_enc.py

```diff
@@ -0,0 +1,43 @@
+import sys
+from unittest import TestCase
+from jaseci.utils.utils import TestCaseHelper
+from fasttext_classifier import serv_actions
+from fastapi.testclient import TestClient
+import os
```

Comment:
let's make sure not import packages that are not being used. I don't think `os` is used in here.